### DISCUSSION TITLE
Publish this template on IIS 10 server 2016

### DISCUSSION THREAD

**Discussion by RaySheikh** [Dotnet-Boxed/Templates#241] [Q&A]
﻿**Which project template is the issue about?**
graphql
...

**Which version of the project template is the issue about?**
latest
...

**Which version of Visual Studio or the dotnet SDK were you using?**
vs 2017 15.9 dotnet sdk 2.1.5
...

Hi, Has anyone able to successfully publish this template on an IIS server. If so can you please share your appsettigs.json and appsettings.production.json file. I'm having issues configuring these files. I'm getting a 404 when I connect using https://localhost:443 but the IIS logs below show its listening.

```
The logs show 
Hosting environment: Production
Content root path: E:\GraphQL\Default Web Site
Now listening on: http://127.0.0.1:14435
Application started. Press Ctrl+C to shut down.
```
Here is my appsettings.production.json

```
{
  "ForwardedHeaders": {
    "AllowedHosts": [
      "example.com"
    ]
  },
  "Kestrel": {
    "Endpoints": {
      "Https": {
        "Url": "http://*:443"
        //"Certificate": {
        //  // TODO
        //}
      },
      "Http": {
        "Url": "http://*:80"
      }
    }
  },
  "Serilog": {
    "MinimumLevel": {
      "Default": "Information",
      "Override": {
        "Microsoft": "Warning",
        "System": "Warning"
      }
    }
  }
}
```

**Comment by RehanSaeed** [comment]
Did you fix this issue? How did you configure IIS? You need to change `example.com` to your own host. Also, fill in a certificate where it says `TODO`.

**Comment by RehanSaeed** [comment]
Feel free to reopen if you have more information.
